extensions/ipynb/src/notebookAttachmentCleaner.ts

```diff
@@ -0,0 +1,191 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { clearTimeout, setTimeout } from 'timers';
+
+/*	Format:
+ *	attachmentCache : { notebookUri : { cellFragment : { filename : { mime : b64 } } } }
+ */
+const attachmentCache: any = {};
```

Comment:
Make sure this has proper typings instead of `any`

Reply:
Also instead of using a global that is shared between functions, can you encapsulate all this into a class?